py3status/module.py

```diff
@@ -324,8 +325,24 @@ def run(self):
                         syslog(LOG_INFO,
                                'method {} returned {} '.format(meth, result))
                 except Exception:
-                    err = sys.exc_info()[1]
-                    msg = 'user method {} failed ({})'.format(meth, err)
+                    try:
+                        # Try to get nice feedback info
+                        # we need to make sure to delete tb even if things go
+                        # wrong.
+                        exc_type, exc_obj, tb = sys.exc_info()
+                        stack = extract_tb(tb)
+                        filename = os.path.basename(stack[-1][0])
+                        line_no = stack[-1][1]
+                        msg = '{}, instance, {}, user method {}, failed ({}) line {}'.format(
```

Comment:
There's an extra comma after `instance` imho should be:
`msg = '{}, instance {}, user method {}, failed ({}) line {}'.format(`